web-app/HeimScripts/heatmap/run.R

```diff
@@ -157,7 +157,7 @@ addStats <- function(df, sorting, ranking, max_rows) {
     df["SD"]           <- sdses
     df["SIGNIFICANCE"] <- rankingScore
 
-    if (useLimma && validLimmaMeasurements) {
+    if (validLimmaMeasurements) {
```

Comment:
applyRanking is now done only if input is valid for limma? What if useLimma is false?